it/service/NotificationViewedMongoRepositoryISpec.scala

```diff
@@ -0,0 +1,73 @@
+
+package service
+
+import helpers.{AssertionHelpers, IntegrationSpec}
+import play.api.test.FutureAwaits
+import repositories.{NotificationViewedMongoRepositoryImpl, StatusNotification, ViewedStatus}
```

Comment:
Unused import present showing as warning when running it tests ... StatusNotification

Reply:
Applied commit to cover this off